src/Fhir.Anonymizer.Core/Processors/KeepProcessor.cs

```diff
@@ -1,10 +1,11 @@
-﻿using Hl7.Fhir.ElementModel;
+﻿using Fhir.Anonymizer.Core.Models;
+using Hl7.Fhir.ElementModel;
 
 namespace Fhir.Anonymizer.Core.Processors
 {
     public class KeepProcessor: IAnonymizerProcessor
     {
-        public void Process(ElementNode node)
+        public void Process(ElementNode node, AnonymizationStatus status)
```

Comment:
More looks like we are trying to use a input parameter to return some result. Could we change the interface to something like:
public ProcessResult Process(ElementNode node)? And merge result outside of processor?

Reply:
Yes, it is fixed in the update.